test/e2e/specs/liquidation-reconstitution.spec.js

```diff
@@ -435,25 +448,69 @@ describe('Wallet App Test Cases', () => {
 
       // Tests ran fine locally but failed in CI. Updating a3p container replicated failure locally. Tests pass with older container version.
       // UNTIL: a3p container compatibility is resolved.
-      it('should wait and verify vaults are liquidated', () => {
-        cy.skipWhen(AGORIC_NET === networks.LOCAL);
 
-        cy.contains(/Collateral left to claim/, {
-          timeout: LIQUIDATED_TIMEOUT,
-        });
-        cy.contains(/3.42 ATOM/);
-        cy.contains(/3.07 ATOM/);
-        cy.contains(/2.84 ATOM/);
+      it('should verify 2 vaults are reconstituted', () => {
+        cy.skipWhen(AGORIC_NET === networks.LOCAL);
+        cy.contains(
+          /Please increase your collateral or repay your outstanding IST debt./,
+          { timeout: LIQUIDATING_TIMEOUT },
+        );
       });
 
+      it(
+        'should verify a vault is liquidated',
+        {
+          defaultCommandTimeout: DEFAULT_TIMEOUT,
+          taskTimeout: DEFAULT_TASK_TIMEOUT,
+        },
+        () => {
+          cy.skipWhen(AGORIC_NET === networks.LOCAL);
+
+          cy.contains(/Collateral left to claim/, {
+            timeout: LIQUIDATED_TIMEOUT,
+          });
+          cy.contains(/0.00 ATOM/);
+        },
+      );
+
       it('should verify the value of collateralAvailable from the CLI successfully', () => {
         cy.skipWhen(AGORIC_NET === networks.LOCAL);
 
         const propertyName = 'book0.collateralAvailable';
-        const expectedValue = '9.659301 ATOM';
+        const expectedValue = '31.414987 ATOM';
 
         cy.verifyAuctionData(propertyName, expectedValue);
       });
+
+      it('should close the 100 debt vault and approve the transaction successfully', () => {
+        cy.skipWhen(AGORIC_NET === networks.LOCAL);
+        const regexVault100 = new RegExp('100(\\.\\d+)?');
+        cy.contains(regexVault100, { timeout: LIQUIDATING_TIMEOUT }).click();
+        cy.contains('Close Out Vault').click();
+        cy.contains('button.bg-interPurple', 'Close Out Vault').click();
```

Comment:
we should avoid using css classes in our selectors, since they can be fragile

Reply:
This happens when we close a vault. And we have two `Close Out Vault` buttons on the screen.